brax/envs/to_torch.py

```diff
@@ -0,0 +1,133 @@
+""" Wrapper around a Brax GymWrapper, that converts its outputs to PyTorch tensors.
+
+This conversion happens direcly on-device, witout the need to move values to the CPU.
+"""
+
+import datetime
+import time
+from collections import abc
+from functools import partial, singledispatch
+from typing import Any, Dict, Mapping, Optional, TypeVar, Union
+
+import gym
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
```

Comment:
Much like we are striving to make tensorflow an optional dependency, let's do the same here.  To that end, can you try something like this:

https://github.com/google/brax/blob/main/brax/io/export.py#L24

Just so the user gets a more informative message if the import fails.  Thanks.